swaybar/i3bar.c

```diff
@@ -104,6 +104,7 @@ static bool i3bar_parse_json(struct status_line *status, const char *text) {
 			json_object_get_int(border_right) : 1;
 		wl_list_insert(&status->blocks, &block->link);
 	}
+  json_object_put(results);
```

Comment:
Wrong indentation.

Reply:
In a single line commit I was able to get an issue... that's a new mark in my life :joy: 

Fixed it, I have my vim configured to use spaces and not tabs. Got it to use a tab now.